scripts/index.js

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+import { Calculator } from "./calculator.js";
+
+// Initialize the calculator.
+let calc = new Calculator("display-text", "control");
```

Comment:
We can be specific about the variable name.

```suggestion
let calculator = new Calculator("display-text", "control");
```

Reply:
Fixed it.
https://github.com/dhruvdhaduk-simform/scientific-calc/pull/3/commits/169d2f47b90ab568975b7e9b35d91ae50c3b4787